### PR TITLE
proper oracle db reset support

### PR DESCRIPTION
**Pull request by akostadinov**
Actually password is hardcoded because I cou;dn't think of a good config option for that. I'm open for suggestions!

### CODE REVIEW

File: bin/porta [hunk 1]
```diff
@@ -607,8 +614,10 @@ module Porta
 
   class ResetCommand < CommandRunner
     def run
+      env_vars = oracle? ? {"ORACLE_SYSTEM_PASSWORD" => "threescalepass"} : {}
+
       system('redis-cli flushall') || raise("redis-cli execution failed")
-      exec_in_porta { "bundle exec rails db:reset MASTER_PASSWORD=#{options[:master_password]} USER_PASSWORD=#{options[:admin_password]} ADMIN_ACCESS_TOKEN=#{options[:admin_access_token]} APICAST_ACCESS_TOKEN=#{options[:apicast_access_token]}" } || raise("db:reset in Porta failed")
+      exec_in_porta(env_vars) { "bundle exec rails db:reset MASTER_PASSWORD=#{options[:master_password]} USER_PASSWORD=#{options[:admin_password]} ADMIN_ACCESS_TOKEN=#{options[:admin_access_token]} APICAST_ACCESS_TOKEN=#{options[:apicast_access_token]}" } || raise("db:reset in Porta failed")
       exec_in_zync { "bundle exec rails db:reset" } || raise("db:reset in Zync failed")
```
Comment (jsmadis) [inline bin/porta]: I have a better option for it:

```ruby
  class ResetCommand < CommandRunner
    def envs
      vars = {
        'MASTER_PASSWORD' => options[:master_password],
        "USER_PASSWORD" => options[:admin_password],
        "ADMIN_ACCESS_TOKEN" => options[:admin_access_token],
        "ZYNC_AUTHENTICATION_TOKEN" => options[:zync_authentication_token],
        "ZYNC_ENDPOINT" => options[:zync_endpoint],
        "APICAST_ACCESS_TOKEN" => options[:apicast_access_token],
      }
      vars['ORACLE_SYSTEM_PASSWORD'] = "threescalepass" if oracle?
    vars
    end

    def run
      system('redis-cli flushall') || raise("redis-cli execution failed")
      exec_in_porta(envs) { "bundle exec rails db:reset" } || raise("db:reset in Porta failed")
      exec_in_zync { "bundle exec rails db:reset" } || raise("db:reset in Zync failed")
    end
  end
```

Comment (akostadinov) [reply bin/porta]: Looks good. But why all the other variables? It worked in the past without them.

Comment (guicassolato) [reply bin/porta]: In spite of differences between environment variables and Rake arguments and how bundler will handle those, both solutions should work.

I'll let you guys decide which one you prefer and I'll then merge whatever you've chosen.

**Review by guicassolato — APPROVED** [review]

File: bin/porta (hunk 1 above)
Comment (jsmadis) [reply bin/porta]: > But why all the other variables? 
You can use it without them. I copied my implementation (I don't have time to make it PR ready yet) 

> It worked in the past without them.
@akostadinov  A lot of things are not working and you don't know about it until you need them :D

**Review by jlledom — APPROVED** [review]

**Pull request merged**